source/sdk/lang/Exception.ooc

```diff
@@ -1,397 +1,395 @@
+			message = message append(':') append(x)
+		} else message = x
+	}
 }
 
 OutOfBoundsException: class extends Exception {
-    init: func (=origin, accessOffset: SizeT, elementLength: SizeT) {
-        init(accessOffset, elementLength)
-    }
-    init: func ~noOrigin (accessOffset: SizeT, elementLength: SizeT) {
-        message = "Trying to access an element at offset %d, but size is only %d!" format(accessOffset,elementLength)
-    }
+	init: func (=origin, accessOffset, elementLength: SizeT) {
+		init(accessOffset, elementLength)
+	}
+	init: func ~noOrigin (accessOffset, elementLength: SizeT) {
+		message = "Trying to access an element at offset %d, but size is only %d!" format(accessOffset, elementLength)
+	}
 }
 
 OutOfMemoryException: class extends Exception {
-    init: func (=origin) {
-        init()
-    }
-    init: func ~noOrigin {
-        message = "Failed to allocate more memory!"
-    }
+	init: func (=origin) {
+		init()
+	}
+	init: func ~noOrigin {
+		message = "Failed to allocate more memory!"
+	}
 }
 
 /* -------- Signal / exception catching ---------- */
 
 version ((linux || apple) && !android) {
-    _signalHandler: func (sig: Int) {
-        message := match sig {
-            case SIGHUP   => "(SIGHUP) terminal line hangup"
-            case SIGINT   => "(SIGINT) interrupt program"
-            case SIGILL   => "(SIGILL) illegal instruction"
-            case SIGTRAP  => "(SIGTRAP) trace trap"
-            case SIGABRT  => "(SIGABRT) abort program"
-            case SIGFPE   => "(SIGFPE) floating point exception"
-            case SIGBUS   => "(SIGBUS) bus error"
-            case SIGSEGV  => "(SIGSEGV) segmentation fault"
-            case SIGSYS   => "(SIGSYS) non-existent system call invoked"
-            case SIGPIPE  => "(SIGPIPE) write on a pipe with no reader"
-            case SIGALRM  => "(SIGALRM) real-time timer expired"
-            case SIGTERM  => "(SIGTERM) software termination signal"
-            case          => "(?) unknown signal %d" format(sig)
-        }
-
-        stderr write(message). write('\n')
-
-        // try to display a stack trace.
-        stderr write(Exception getCurrentBacktrace())
-
-        exit(sig)
-    }
+	_signalHandler: func (sig: Int) {
+	message := match sig {
+		case SIGHUP => "(SIGHUP) terminal line hangup"
+		case SIGINT => "(SIGINT) interrupt program"
+		case SIGILL => "(SIGILL) illegal instruction"
+		case SIGTRAP => "(SIGTRAP) trace trap"
+		case SIGABRT => "(SIGABRT) abort program"
+		case SIGFPE => "(SIGFPE) floating point exception"
+		case SIGBUS => "(SIGBUS) bus error"
+		case SIGSEGV => "(SIGSEGV) segmentation fault"
+		case SIGSYS => "(SIGSYS) non-existent system call invoked"
+		case SIGPIPE => "(SIGPIPE) write on a pipe with no reader"
+		case SIGALRM => "(SIGALRM) real-time timer expired"
+		case SIGTERM => "(SIGTERM) software termination signal"
+		case => "(?) unknown signal %d" format(sig)
+	}
+
+		stderr write(message). write('\n')
```

Comment:
Why isn't this comment gone ? I just updated this few minutes ago, line 300 looks different now. @marcusnaslund  : are you commenting on the PR or on the commit ?